backend/imap/src/main/java/com/fsck/k9/backend/imap/ImapBackendPusher.kt

```diff
@@ -50,6 +50,11 @@ internal class ImapBackendPusher(
     private var currentIdleRefreshMs = 15 * 60 * 1000L
 
     override fun start() {
+        if (!idleRefreshManager.canScheduleTimers()) {
+            Timber.v("Not starting ImapBackendPusher for %s because the app can't schedule timers", accountName)
+            return
+        }
```

Comment:
This will stop updating folders (the logic below), Android 14 users without `SCHEDULE_EXACT_ALARM` permission will not receive new message notifications.